lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -590,7 +595,6 @@ def test_mfe_link_from_about_page(self):
         with self.store.default_store(ModuleStoreEnum.Type.split):
             course = CourseFactory.create()
         CourseEnrollment.enroll(self.user, course.id)
```

Comment:
Why don't you need to login? Is it already done?

Reply:
Some of these user-setup changes were done because of changing from normal users to staff users or similar. But this one :shrug: